### PR TITLE
Minor fennec treat spelling fixes

### PR DESCRIPTION
**Pull request by dwinters99**
Title.

### CODE REVIEW

File: code/modules/food/food/snacks_vr.dm [hunk 1]
```diff
@@ -176,7 +176,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/bugball
 	name = "bugball"
-	desc = "A hard chitin, dont chip a tooth!"
+	desc = "A hard piece of chitin, dont chip a tooth!"
```
Comment (Arokha) [inline code/modules/food/food/snacks_vr.dm:179]: But didn't fix the apostrophe missing in `dont`

File: code/modules/food/food/snacks_vr.dm [hunk 2]
```diff
@@ -250,7 +250,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/lobster
 	name = "raw lobster"
-	desc = "a shifty lobster. You can try eating it, but its shell is extremely tough."
+	desc = "A shifty lobster. You can try eating it, but it's shell is extremely tough."
```
Comment (Arokha) [inline code/modules/food/food/snacks_vr.dm:253]: https://www.merriam-webster.com/words-at-play/when-to-use-its-vs-its

File: code/modules/food/food/snacks_vr.dm [hunk 3]
```diff
@@ -261,7 +261,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/lobstercooked
 	name = "cooked lobster"
-	desc = "a luxurious plate of cooked lobster, its taste accentuated by lemon juice. Reinvigorating!"
+	desc = "A luxurious plate of cooked lobster, it's taste accentuated by lemon juice. Reinvigorating!"
```
Comment (Arokha) [inline code/modules/food/food/snacks_vr.dm:264]: https://www.merriam-webster.com/words-at-play/when-to-use-its-vs-its

File: code/modules/food/food/snacks_vr.dm [hunk 4]
```diff
@@ -277,7 +277,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/cuttlefish
 	name = "raw cuttlefish"
-	desc = "it's an adorable squid! you can't possible be thinking about eating this right?"
+	desc = "It's an adorable squid! you couldn't possibly be thinking about eating this, right?"
```
Comment (ArgobargSoup) [inline code/modules/food/food/snacks_vr.dm:280]: Missed a capitalization!

File: code/modules/food/food/snacks_vr.dm [hunk 5]
```diff
@@ -677,7 +677,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/scorpion_cooked
 	name = "cooked scorpion"
-	desc = "a scorpion baked nice and crispy"
+	desc = "A scorpion. Baked nice and crispy"
```
Comment (ArgobargSoup) [inline code/modules/food/food/snacks_vr.dm:680]: A period at the end, here?

**Pull request merged**